arccos.json

```diff
@@ -23,7 +23,8 @@
             "type": [
                 "number",
                 "null"
-            ]
+            ],
+            "minimum": 0
```

Comment:
Yes, but I wasn't sure how to expose Pi properly in JSON.

Reply:
just as floating point `3.1415...` will be good enough for most purposes I guess

Reply:
Yeah, for most. But now that I'm writing the tests, these edge cases get in the way of that. 

If I use 3.1415, a test with 3.14159 would fail. So I'd need to set it to 3.1416, which is a bit weird. The precision is an example and could apply on any level of precition. Thus I left it open.

Reply:
I added the range to the text for all functions. I hope that makes the ranges clearer without fiddling to much with specific decimal numbers.